src/styles/components/substring-highlight.scss

```diff
@@ -0,0 +1,9 @@
+@import '../colours';
+
+.highlight {
```

Comment:
Do we want that to be part of the design system in general?
```suggestion
.highlight, mark {
```
Would allow us to use `<mark>` anywhere we want (not necessarily with the substring component) and have consistent styling

Reply:
Also might make more sense to move that style into a `styles/common/_mark.scss`

Reply:
We could, although I'm afraid we might lose the link between the component and the style in a refactor, or not load the correct style whenever needed.
In that case I'd put this style as part of the global franklin styles and not in a file that is loaded by this component

Reply:
Right, we crossed our replies, but `styles/common/_mark.scss` approach looks good to me

Reply:
Updated